7_react/react1/week3/todo-app/src/component/TodoContainer.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useEffect } from 'react';
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import 'bootstrap/dist/css/bootstrap.min.css'
+import {FaCalendarAlt} from 'react-icons/fa';
+
+//function for calender icon
+function CustomInput({value,onClick}){
+    return(
+        <div className='input-group'>
```

Comment:
for better User experience I would attach the `onClick={onClick} ` function to this `div`, then you will be able to open the calendar by clicking on the calendar icon :)